core/primer.md

```diff
@@ -33,7 +34,130 @@ associating those declarations with application endpoints. As a result, the
+schemas for systems. Multiple schema registry solutions exist already,
+including [Confluent Schema Registry](https://docs.confluent.io/platform/current/schema-registry/index.html), [Azure Event Hubs Registry](https://learn.microsoft.com/en-us/azure/event-hubs/schema-registry-overview)
+or [Apicurio Registry](https://www.apicur.io/registry/), [AWS Glue Schema Registry](https://docs.aws.amazon.com/glue/latest/dg/schema-registry.html)
+and more.
+
+These schema registries are broker- and therefore vendor-specific and even
+protocol-specific, which causes friction when an event travels through multiple
+brokers. In such cases, clients have to deal with the implementation differences
+across registries, for example when validating event structures, hindering
+interoperability and, therefore, complicating integration scenarios. xRegistry
+aims to address this with an agnostic specification, providing a common base
+across these offerings.
+
+### Versioning
+
+As systems evolve, the endpoints that emit events may change, as may the events
+themselves. Changes can sometimes break consumers making compatibility
+strategies essential to indicate to consumers how they may expect events or
+endpoints to evolve. xRegistry allows indicating a registry’s compatibility
+strategy which is enforced when making changes to the registry, avoiding
+mistakes. The specifications further also enables marking endpoints as
+deprecated, possibly providing an alternative endpoint, making these easier to
+discover for consumers. Finally, event schemas are also version aware, storing
+multiple versions of an event’s data schema.
+
+### Schema validation
+
+The registry contains granular definitions for events defining their schemas,
+which serve as a contract between publishers and consumers. This doesn’t only
+apply to the message payload, but also to the message metadata, allowing
+granular control of required fields on the metadata level. The registry does not
+only serve as a definition of the required schemas, but can also be used to
+enforce it. A registry service could use the registry to perform validation
+on-publish, rejecting any events that don’t adhere to the schema for that
+context, avoiding the problem of poison messages.
+
+### Payload reduction
+
+Schema information enables real-time analysis of incoming data based on its
+contextual meaning, which is why applications often use serialization mechanisms
+which include schema information (such as JSON) in payloads. Having a
+centralized registry, enables applications to rely on lighter weight
+serialization formats, significantly reducing bandwidth, while still allowing
+accurate real-time analysis of incoming data using the registry.
+
+### Schema-based data contract generation
+
+xRegistry provides a specification to manage metadata in files (for example,
+stored next to relevant source code), but the metadata can also be managed in a
```

Comment:
While the rest of this section is true, I think storing the files next to the code is too quickly passed-over. Being able to link the proper version of the model and corresponding code via co-location in git huge. Perhaps we can expand a bit on this (even with just one more sentence) by talking about how much easier it makes it to recreate the state of the system/environment when everything can be stored together w/o complicated tooling or "linking" of data sources.

Reply:
Addressed in [`0b2ba34` (#219)](https://github.com/xregistry/spec/pull/219/commits/0b2ba34b42d3a45b2812af948bf045c6bb7bb9d9)